DllImportGenerator/DllImportGenerator.UnitTests/Compiles.cs

```diff
@@ -159,11 +159,13 @@ public static IEnumerable<object[]> CodeSnippetsToCompile_NoDiagnostics()
             yield return new[] { CodeSnippets.CustomStructMarshallingPinnableParametersAndModifiers };
             yield return new[] { CodeSnippets.CustomStructMarshallingNativeTypePinnable };
             yield return new[] { CodeSnippets.CustomStructMarshallingMarshalUsingParametersAndModifiers };
+            yield return new[] { CodeSnippets.ArrayMarshallingWithCustomStructElement };
         }
 
         public static IEnumerable<object[]> CodeSnippetsToCompile_WithDiagnostics()
         {
             yield return new[] { CodeSnippets.AllSupportedDllImportNamedArguments };
+            yield return new[] { CodeSnippets.ArrayMarshallingWithCustomStructElementWithValueProperty };
```

Comment:
This should go into the non-supported tests. The above snippets are for those we planned on enabling.

Reply:
I was thinking that we do plan on enabling this, just not in the immediate term. This is for only the near-term enabling in v1 scenario, then I'll move this over to the non-supported tests.

Reply:
Yep. I think the calculus has changed how much we are going to be able to enable in v1. I would move it for now.